.github/workflows/build.yml

```
@@ -46,7 +41,12 @@ jobs:
         run: make lint
 
       - name: Run tests
-        run: make test
+        run: |
+          curl -L https://get.gear.rs/vara-testnet-v1.1-x86_64-unknown-linux-gnu.tar.xz -o - | tar xJ
+          chmod +x gear
+          ./gear --tmp --dev &> /dev/null & echo -n $! > gear.pid
+          make test
+          kill `cat gear.pid`
```

```suggestion
        run: make test
```
Since there's no gclient tests, you can leave just `make test` here.